RELEASE.md

```diff
@@ -0,0 +1,17 @@
+Release type: minor
+
+This release adds support for the `graphql-transport-ws` GraphQL over WebSocket
+protocol. Previously Strawberry only supported the legacy `graphql-ws` protocol.
+
+Developers can decide which protocols they want to accept. The following example shows
+how to do so using the ASGI integration. Take a look at our GraphQL subscription
+documentation to learn more.
+
+```python
+from strawberry.asgi import GraphQL
+from strawberry.subscriptions import GRAPHQL_TRANSPORT_WS_PROTOCOL
+from api.schema import schema
+
+
+app = GraphQL(schema, subscription_protocols=[GRAPHQL_TRANSPORT_WS_PROTOCOL])
```

Comment:
could we do this using the SchemaConfig class?

or could we chose it automatically based on the request?

Reply:
> or could we chose it automatically based on the request?

The protocol is automatically choosen based on the request. The `subscription_protocols` option allows you to disable all or individual protocols (eg. if you don't  want to support the legacy protocol because of potentiall security implications etc.).

Reply:
> could we do this using the SchemaConfig class?

we could, but there are other subscription related options for both protocols which we should probably move then as well, making this a breaking change. Also I think these options are more related to the view than the schema, so I'm not sure we want them in the schema config 🤔

Reply:
oh, didn't realise that, awesome!

Reply:
> we could, but there are other subscription related options for both protocols which we should probably move then as well, making this a breaking change. Also I think these options are more related to the view than the schema, so I'm not sure we want them in the schema config 🤔

makes sense! let's keep them on the view then 😊